sections/04_study.md

```diff
@@ -138,7 +141,7 @@ instructive in that clever sources of data, not just more descriptive models,
 are still critical.
 
 We already understand how mis-splicing of a single gene can cause diseases such
```

Comment:
@bdo311 The authors of the review use the language that splicing-related mutations can cause disease, so I'm comfortable with `cause` here.  If you still object, I don't feel strongly about it so we could make the change.